reproducibility_project/src/engines/mcccs/project.py

```diff
@@ -23,40 +23,77 @@ def __init__(self):
 ex = Project.make_group(name="ex")
 
 
+def mc3s_exec():
+    """Return the path of MCCCS-MN executable."""
+    return "/home/rs/group-code/MCCCS-MN-7-20/exe-8-20/src/topmon"
+
+
 def get_system(job):
     """Return the system (mbuild filled_box) for a particular job."""
-    molecule = job.sp.molecule
-    import warnings
+    import sys
 
-    import mbuild as mb
+    import foyer
+    from constrainmol import ConstrainedMolecule
 
-    from reproducibility_project.src.utils.forcefields import load_ff
+    sys.path.append(Project().root_directory())
+    from src.utils.forcefields import load_ff
```

Comment:
check if this is still necessary after the pip install update.

Reply:
Works without adding the path.Thanks